tests/api/worlds/camera_color.wbt

```diff
@@ -40,15 +40,17 @@ Robot {
   children [
     Camera {
       zoom Zoom {
-        minFieldOfView 1.56
         maxFieldOfView 1.58
+        minFieldOfView 1.56
       }
     }
     TestSuiteEmitter {
     }
   ]
   controller "camera_checker"
-  controllerArgs "camera_color"
+  controllerArgs [
+    "camera_color"
+  ]
```

Comment:
Just for the note, we discussed this case on Monday, and it seems Webtos is saving these MFString fields on a new line.